build.gradle

```diff
@@ -2,15 +2,15 @@
 apply plugin: 'io.codearte.nexus-staging'
 
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.6.21'
```

Comment:
Change made by Upgrade Assistant

> Upgrade Gradle plugins
> 
> Some Gradle plugins in the project use interfaces which are no longer supported in version 7.5 (or later) of Gradle and version 7.4.0 (or later) of Android Gradle Plugin.
The following Gradle plugin versions will be updated:
Update version of org.jetbrains.kotlin:kotlin-gradle-plugin to 1.6.21

I'll update Kotlin to a newest version in separate PR.